closed/src/java.base/share/native/libjncrypto/NativeCrypto.c

```diff
@@ -38,10 +39,14 @@
 #include "jdk_crypto_jniprovider_NativeCrypto.h"
 #include "NativeCrypto_md.h"
 
-#define OPENSSL_VERSION_1_0 "OpenSSL 1.0."
-#define OPENSSL_VERSION_1_1 "OpenSSL 1.1."
+#define OPENSSL_VERSION_CODE(major, minor, fix, patch) (((major) << 28) | ((minor) << 20) | ((fix) << 12) | (patch))
```

Comment:
Please split this as in my earlier suggestion.
Of course, all uses of `OPENSSL_VERSION_CODE` now need four arguments.

Reply:
Updated.